bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -0,0 +1,147 @@
+#3dimensional_kde 
+#python3 pycbc_template_kde_calc.py --template-file /home/praveen.kumar/focused_search/kde/sbbh2.hdf --signal-file /home/praveen.kumar/focused_search/kde/3OGC_PEtable.txt_ifar_gt0.5.csv     --make-signal-kde    --verbose
+
+import numpy as np
+import operator
+import pycbc
+import h5py
+import pycbc.conversions as convert
+from utils_awkde import *
+from awkde import GaussianKDE
+from sklearn.model_selection import KFold
+import argparse
+import logging
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--signal-file', required=True)
+parser.add_argument('--template-file', required=True)
+parser.add_argument('--which-kde', choices=['signal','template'])
+parser.add_argument('--make-signal-kde', action='store_true')
+parser.add_argument('--make-template-kde', action='store_true')
+parser.add_argument('--verbose', action='count')
+args = parser.parse_args()
+pycbc.init_logging(args.verbose)
+
+def kde_awkde(x, x_grid, alp=0.5, gl_bandwidth='silverman', ret_kde=False):
+    kde = GaussianKDE(glob_bw=gl_bandwidth, alpha=alp, diag_cov=True)
+    kde.fit(x)#[:, np.newaxis])
```

Comment:
It looks like the second half of this line is commented out, can you remove the commented portion